### PR TITLE
Use originalFSPath in terminal

### LINKED ISSUE #88906: NPM Scripts: wrong drive letter casing on Windows
Issue Type: <b>Bug</b>

On Windows, in a Vue.js project:
Start "npm run serve" from the built in "NPM Scripts" view. Output:

> Executing task: npm run serve <
> my-app@1.0.0 serve c:\Users\dan\...
> vue-cli-service serve
> INFO  Starting development server...

The problem is the lowercase "c:\", which in my project will lead to these warnings:
> There are multiple modules with names that only differ in casing.."

Typing "npm run serve" in the built in terminal works fine, so it must be a bug within the "NPM Scripts" view.

VS Code version: Code 1.41.1 (26076a4de974ead31f97692a0d32f90d735645c0, 2019-12-18T14:58:56.166Z)
OS version: Windows_NT x64 10.0.18362

### PR DESCRIPTION
**Pull request by alexr00**
Fixes https://github.com/microsoft/vscode/issues/88906

### CODE REVIEW

**Comment by Tyriar** [comment]
Isn't this meant to be handled by https://github.com/microsoft/vscode/blob/8066726b2752fee926dbc4d4b3ba1a9a3968c09d/src/vs/workbench/contrib/terminal/common/terminalEnvironment.ts#L227-L234? Is the problem tasks is seeing that `_sanitizeCwd` isn't called at this line?

https://github.com/microsoft/vscode/blob/8066726b2752fee926dbc4d4b3ba1a9a3968c09d/src/vs/workbench/contrib/terminal/common/terminalEnvironment.ts#L186-L190

**Comment by alexr00** [comment]
That would would also fix it.

**Comment by alexr00** [comment]
I'll test it and update the PR.

File: src/vs/workbench/contrib/terminal/common/terminalEnvironment.ts [hunk 1]
```diff
@@ -184,7 +185,7 @@ export function getCwd(
 	logService?: ILogService
 ): string {
 	if (shell.cwd) {
-		const unresolved = (typeof shell.cwd === 'object') ? shell.cwd.fsPath : shell.cwd;
+		const unresolved = (typeof shell.cwd === 'object') ? originalFSPath(shell.cwd) : shell.cwd;
```
Comment (Tyriar) [inline src/vs/workbench/contrib/terminal/common/terminalEnvironment.ts]: Maybe just use originalFSPath here? I'm worried about regressions in the latter parts.

Comment (alexr00) [reply src/vs/workbench/contrib/terminal/common/terminalEnvironment.ts]: I changed to just use _sanitizeCwd on the early return instead.

**Comment by alexr00** [comment]
`_sanitizeCwd` also solves the issue.

**Review by Tyriar — APPROVED** [review]

**Pull request merged**